components/teach/Courses.vue

```diff
@@ -2,21 +2,7 @@
   <bx-skeleton-text v-if="pending" type="heading" />
   <SectionContainer v-if="courses.length">
     <h2>Courses</h2>
-    <div class="courses-container">
-      <MediaCard
-        v-for="course in courses"
-        :key="course.id"
-        :label="course.category"
-        :title="course.title"
-        :media="courseStore.getCourseMediaUrl(course)"
-      >
-        <template v-if="course.description" #body>
-          <p>
-            {{ course.description }}
-          </p>
-        </template>
-      </MediaCard>
-    </div>
+    <CourseCardGrid :courses="courses || []" />
```

Comment:
```suggestion
    <CourseCardGrid :courses="courses" />
```

`courses` is already an empty array by default in the `store/course/store.ts`